R/CohortConstruction.R

```diff
@@ -458,12 +461,31 @@ getInclusionStats <- function(exportFolder,
     ))
   }
   if (nrow(subset) > 0) {
+    if (is.null(inclusionStatisticsFolder)) {
```

Comment:
@anthonysena I'd appreciate it if you approve of this method for extracting the cohort stats data; it outputs the csv files to a temp folder then imports them in to cohort diagnostics.

This is a bit messy but I think its ok for now because it minimises changes in the cohort diagnostics code base. However, I think a future version of CohortGenerator we should include functionality to export the CohortStats to data frames directly.

Reply:
Hey @azimov this looks good from my review. Just a question that I had after reading through the code - CohortGenerator is invoked under the conditions that there is a non-zero set of cohorts and a null `inclusionStatisticsFolder`. Since this appears to be a private method, is there a condition where the `inclusionStatisticsFolder` would be non-null and skip the retrieval? 

On your other point: I agree we can add a method to CohortGenerator that returns the cohort stats as a data frame so you don't have to do this two-step on the file system.

Reply:
Thanks @anthonysena yes, there is the case where `inclusionStatisticsFolder` can be set by the user; this would be the old method of calling `runCohortDiagnostics` which will be retired in a future version but is maintained here for backwards compatibility.